File: 1_4/1task.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = 26;
        if (N > 26) {
            System.out.println("Incorrect data");
        } 
        else {
            for (int i = N; i > 0; i--) {
                for (int k = N; k >= i; k--) {
                    char Str = (char) ('A' + k - 1);
                    System.out.print(Str + " ");
                }
                System.out.println();
            }
        }
    }
}

File: 1_4/1task.go
package main

import "fmt"

func main() {
    N := 4
    if N > 26 {
        fmt.Println("Incorrect data")
    } else {
        for i := N; i > 0; i-- {
            for k := N; k >= i; k-- {
                Str := 'A' + k - 1
                fmt.Printf("%c ", Str)
            }
            fmt.Println()
        }
    }
}